### PR TITLE
Font fallback

### PR DESCRIPTION
**Pull request by fundies**

### CODE REVIEW

**Comment by fundies** [comment]
replaces #1635. @RobertBColton this needs testing thanks

File: ENIGMAsystem/SHELL/Universal_System/Extensions/ttf/ttf.cpp [hunk 1]
```diff
@@ -37,32 +45,122 @@ class FontManager {
...
+      n = name + "-" + s2 + ".ttf";
+      possibleNames.push_back(n);
+      std::transform(n.begin(), n.end(), n.begin(), ::tolower);
+      possibleNames.push_back(n);
+      
+      n = name + "-" + s3 + ".ttf";
+      possibleNames.push_back(n);
+    };
+    
+    // Win32 Best Match
```
Comment (fundies) [inline ENIGMAsystem/SHELL/Universal_System/Extensions/ttf/ttf.cpp]: The file names on win32 font match the names of the font for whatever reason so we need some kind of map for that too

**Comment by fundies** [comment]
I would prefer to loop know font dirs and extract the names out of the ttfs or something but this a pita without std::filesystem

File: ENIGMAsystem/SHELL/Universal_System/Extensions/ttf/ttf.cpp [hunk 2]
```diff
@@ -37,32 +45,134 @@ class FontManager {
...
+};
+
 FT_Library FontManager::library;
 static bool FreeTypeAlive = FontManager::Init();
 
+std::vector<std::string> fontSearchPaths;
+
+// TODO: Fill in this map in
+// https://docs.microsoft.com/en-us/typography/fonts/windows_10_font_list
+const std::map<std::string, std::string> msFontsNames = {
```
Comment (fundies) [inline ENIGMAsystem/SHELL/Universal_System/Extensions/ttf/ttf.cpp]: @RobertBColton it is possible to generate this map on windows using win32 crap but we would want access to it on linux and os x as well as it is possible to install msfonts on linux. I'd appreciate it if you could fill out that map cause I cba. I linked the reference above but you could also write a 4 line app to generate that map too

**Comment by RobertBColton** [comment]
There's #1849 now.

**Pull request closed without merging**